src/common/gui/CSurgeSlider.cpp

```diff
@@ -777,11 +780,16 @@ CMouseEventResult CSurgeSlider::onMouseUp(CPoint& where, const CButtonState& but
       
       edit_value = nullptr;
 
-
-      if( resetPosition )
+      if (resetPosition &&
+          startPosition != currentPosition)
```

Comment:
This comparison could be a threshold, but after using it like this for a while, it feels most natural like this.